pyramid_oereb/standard/pyramid_oereb.yml.mako

```diff
@@ -84,6 +84,11 @@ pyramid_oereb:
     # The headers sent to the print
     headers:
       Content-Type: application/json; charset=UTF-8
+    furtherInfromationText:
+      de: [Hier sollten die weiteren Informationen erfasst werden.]
```

Comment:
Your sure about the [], It will mean that we get an array with the words...

Reply:
The "Further_Information_Text" attribute is in the JSON when I call the service with "getspec". This means to me, that there is wrong something else. Do I need to configure the new parameter somwhere else?
@voisardf: Maybe you can help out here?

Reply:
@vvmruder I don't understand what's wrong ...

Reply:
Öhhh... You saw the error I posted above?

I think this is caused by mapfish print? What is normally causing this error?

Reply:
No idea about the error, for me everything looks correct ...

Reply:
what about the brackets? could that be the issue in the yaml?